src/main/java/Shrek.java

```diff
@@ -0,0 +1,141 @@
+import java.util.Scanner;
+
+public class Shrek {
+    private static final int NUMBER_OF_INPUT = 100;
+    private static final UserContent[] lists = new UserContent[NUMBER_OF_INPUT];
+    private static int listIndex = 1;
+    private static final String LINE = PrintStrings.LINE;
+    private static final int INDEX_OF_TASK_CONTENT = 0;
+    private static final int INDEX_OF_TASK_NAME = 1;
+    private static final int INDEX_OF_TASK_COMMAND = 0;
+    private static final int INDEX_OF_TASK_INPUT = 1;
```

Comment:
Nice job for keeping all the constants in upper cases and using "static final" for constants